composer.json

```
@@ -70,6 +70,7 @@
         "drupal/migrate_plus": "5.1",
         "drupal/migrate_tools": "5.0",
         "drupal/paragraphs": "1.12",
+        "drupal/paragraphs_admin": "^1.2",
```

Can we pin this to a specific version?  That's our convention for this repo (in order to try to keep our builds as deterministic as possible).
```suggestion
        "drupal/paragraphs_admin": "1.2",
```